<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="labelSaveFile">
           <property name="text">
            <string>Save File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineSaveFile"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labeln_session">
           <property name="text">
            <string>n_session</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linen_session"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="labeln_run">
           <property name="text">
            <string>n_run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linen_run"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="labeln_trial">
           <property name="text">
            <string>n_trial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linen_trial"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="labelDataPeriod">
           <property name="text">
            <string>Data Period(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineDataPeriod"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="labelTrialLength">
           <property name="text">
            <string>Trial Length(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineTrialLength"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="cBoxLeftHand">
         <property name="text">
          <string>Left Hand</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxRightHand">
         <property name="text">
          <string>Right Hand</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxTongue">
         <property name="text">
          <string>Tongue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxLeftFoot">
         <property name="text">
          <string>Left Foot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxRightFoot">
         <property name="text">
          <string>Right Foot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxRest">
         <property name="text">
          <string>Rest</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnConfirm">
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
